services/wallet/router/router_v2.go

```diff
@@ -33,6 +34,7 @@ var (
 )
 
 type RouteInputParams struct {
+	Uuid                 int64                   `json:"uuid"`
```

Comment:
we normally use `string` for uuid in other parts of the app (https://en.wikipedia.org/wiki/Universally_unique_identifier)